ci/disallow_c_comments_in_migrations.sh

```diff
@@ -5,4 +5,5 @@ set -eu
 # We do not use c-style comments in migration files as the stripped
 # zero-length migration files cause warning during packaging
 # See #3115 for more info
-find src/backend/distributed/sql/*.sql -print0 | xargs -0 sed -i 's#/\*\(.*\)\w*\*/#--\1#g'
+find src/backend/distributed/sql/*.sql -print0 | xargs -0 sed -i 's#^\s*/\*\(.*\)\w*\*/#--\1#g'
+find src/backend/distributed/sql/*.sql -print0 | xargs -0 sed -i 's#^\s*\*\/\s*##g'
```

Comment:
What is this regex supposed to do?

Reply:
I think you misunderstood my regex:
`sed s#^ *//#--#g`

It was supposed to remove `//` with `--`. Thinking a bit more, I actually think it's good to keep the whitespace at the start of a comment. Because it might be used in a function, and removing it would make the indentation weird.

Reply:
I updated the script to include 4 different patterns.

- `/*` -> `--` 
- `*/` -> `` (empty string)
- ` *` -> `--`
- `//` -> `--`